src/app/components/RelatedContentSection/RelatedContentItem/index.tsx

```diff
@@ -22,12 +22,39 @@ const RelatedContentItem = forwardRef<HTMLDivElement, RelatedContentItemProps>(
   ({ item, ariaLabelledBy, eventTrackingData = null }, viewRef) => {
     if (!item || isEmpty(item)) return null;
 
-    const headline = pathOr<string>(
+    const headlineFirst = pathOr<string>(
+      '',
+      ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+      item,
+    );
+    const headlineSecond = pathOr<string>(
       '',
       ['model', 'blocks', 1, 'model', 'blocks', 0, 'model', 'text'],
       item,
     );
-    const assetUri = path<string>(
+
+    const headline = headlineFirst || headlineSecond;
```

Comment:
Out of interest, why would there be a 'first headline' and a 'second headline'? (and same with AssetUri)